src/transformers/convert_slow_tokenizer.py

```diff
@@ -578,23 +580,65 @@ def tokenizer(self, proto):
         unk_id = self.unk_id(proto)
 
         if model_type == 1:
-            tokenizer = Tokenizer(Unigram(vocab_scores, unk_id))
+            import tokenizers
+
+            if version.parse(tokenizers.__version__) >= version.parse("0.14.0"):
```

Comment:
pretty sure the versions of `transformers` already require `    "tokenizers>=0.19,<0.20"` so this is always true!

Reply:
Oh yes, you're right! Removed :)